topology/University of California San Diego/UCSD CMS Tier2/UCSDT2_downtime.yaml

```diff
@@ -729,3 +729,58 @@
   Services:
   - GridFtp
 # ---------------------------------------------------------
+- Class: SCHEDULED
+  ID: 344439216
+  Description: Down to work on cooling system
+  Severity: Outage
+  StartTime: Sep 26, 2019 13:30 +0000
+  EndTime: Sep 27, 2019 13:30 +0000
+  CreatedTime: Sep 25, 2019 15:38 +0000
+  ResourceName: UCSDT2-A
+  Services:
+  - CE
+# ---------------------------------------------------------
+- Class: SCHEDULED
+  ID: 344439217
+  Description: Down to work on cooling system
+  Severity: Outage
+  StartTime: Sep 26, 2019 13:30 +0000
+  EndTime: Sep 27, 2019 13:30 +0000
+  CreatedTime: Sep 25, 2019 15:38 +0000
+  ResourceName: UCSDT2-B
+  Services:
+  - CE
+# ---------------------------------------------------------
+- Class: SCHEDULED
+  ID: 344439218
+  Description: Down to work on cooling system
+  Severity: Outage
+  StartTime: Sep 26, 2019 13:30 +0000
+  EndTime: Sep 27, 2019 13:30 +0000
+  CreatedTime: Sep 25, 2019 15:38 +0000
+  ResourceName: UCSDT2-C
+  Services:
+  - CE
+# ---------------------------------------------------------
+- Class: SCHEDULED
+  ID: 344439219
+  Description: Down to work on cooling system
+  Severity: Outage
+  StartTime: Sep 26, 2019 13:30 +0000
+  EndTime: Sep 27, 2019 13:30 +0000
+  CreatedTime: Sep 25, 2019 15:38 +0000
+  ResourceName: UCSDT2-D
+  Services:
+  - CE
+# ---------------------------------------------------------
+- Class: SCHEDULED
+  ID: 344439220
+  Description: Down to work on cooling system
+  Severity: Outage
+  StartTime: Sep 26, 2019 13:30 +0000
+  EndTime: Sep 27, 2019 13:30 +0000
+  CreatedTime: Sep 25, 2019 15:38 +0000
+  ResourceName: UCSDT2-SE
```

Comment:
There's no "UCSDT2-SE" but there are "UCSDT2-SE1" and "UCSDT2-SE3" so this should be one of those.

Reply:
Sorry should have been UCSDT2-SE1

Reply:
Can you push the change or should I make it?

Reply:
i just fixed it in a new commit on my branch, was that enough?

Reply:
Yep, looks good.